### ISSUE TITLE
prettyCox

### ISSUE THREAD

**Issue by choileena** [oncoestimand/princ_strat_drug_dev#1]
The function prettyCox does not work as some required ones are not defined (e.g., disp, displayCI).

**Comment by numbersman77** [comment]
True! I have now made visible the setup chunk, I hope it works now!
